### ISSUE TITLE
importScripts not recognised as built-in function

### ISSUE THREAD

**Issue by GenaBitu** [eslint/eslint#11180]
**What rule do you want to change?**
`no-undef`

**Does this change cause the rule to produce more or fewer warnings?**
fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**
New default behaviour - the function [importScripts](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts) will be recognised as browser built-in (not sure that is the correct term, I just mean a function provided by the browser)

**Please provide some example code that this change will affect:**

```js
importScripts('foo.js');
```

**What does the rule currently do for this code?**
Report `importScripts` as an undefined function

**What will the rule do after it's changed?**
Not report it

**Are you willing to submit a pull request to implement this change?**
Haven't seen the source code, might.

**Comment by g-plane** [comment]
You should enable `worker` env in ESLint config.

**Comment by GenaBitu** [comment]
Derp. You're right, that solved it. Thanks
